corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -134,6 +134,7 @@ def __call__(self, fn):
     'commcaresession': dict(id='commcaresession', src='jr://instance/session'),
     'registry': dict(id='registry', src='jr://instance/remote'),
     'results': dict(id='results', src='jr://instance/remote'),
+    'search_cases': dict(id='search_cases', src='jr://instance/selected_cases'),
```

Comment:
Based on the Formplayer PRs I think this ID should be `selected_cases`. I also think that makes more sense since it's not only case search that would support multi-select.